specification/resources/resource-manager/readme.md

```diff
@@ -83,6 +83,7 @@ input-file:
 - Microsoft.Authorization/stable/2020-09-01/policyAssignments.json
 - Microsoft.Authorization/stable/2020-09-01/policyDefinitions.json
 - Microsoft.Authorization/stable/2020-09-01/policySetDefinitions.json
+- Microsoft.Authorization/preview/2020-07-01-preview/policyExemptions.json
```

Comment:
Could we do not add this preview swagger into this stable tag?
Adding this swagger to this tag will make this tag a preview tag, which will change how things work in go SDK, also the root cause of the go SDK generation failure.

Reply:
@ArcturusZhang These resource types share the same poliyClient.cs when generating .NET SDK. If I remove the policyExemption.json, I don't think the .NET SDK will work. We plan to publish the stable version for policyExemption at around September. We will replace the preview policyExemption with a stable policyExemption then. Is that OK, we generate the go SDK then?

Reply:
We need to work around the CI failure on this PR. The CI is failing because this tag previously in go SDK changed to preview but its output-folder does not change. 
To resolve it, we need to change the output-folder in `readme.go.md` for this tag (adding a `preview` segment after the `services`). It is fine since we have never released this yet. And when exemption gets GA, we will have to change it from preview to stable again.
Could you please change it in `readme.go.md` to resolve the CI failure?

Reply:
@ArcturusZhang , thank you for the detail. I just updated the PR to address it.